cluster-provision/images/centos8-realtime/publish.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker tag quay.io/jordigilh/centos8-realtime-container-disk:latest quay.io/kubevirtci/centos8-realtime-container-disk:latest
```

Comment:
I remember that quay will "garbage collect" untagged images after a while. Is that what we want? Otherwise we might consider using a tag like we do [in project-infra](https://github.com/kubevirt/project-infra/blob/main/images/publish_image.sh#L59). WDYT?

Reply:
Sounds good. This image is meant to be used only for the CI/CD job, not for public consumption, so it makes sense to tag it specifically :smile: